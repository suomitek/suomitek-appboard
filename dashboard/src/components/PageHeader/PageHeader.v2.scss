.kubeapps-header {
  width: 100%;
  height: 4rem;
  padding: 0.6rem;
  border-bottom: 2px solid #f1f1f1;
  background-color: white;

  img {
    max-height: 3rem;
    margin: 0.2rem 0 0.2rem 0.6rem;
  }
  .kubeapps-header-content {
    width: 100%;
    .kubeapps-title-block {
      display: flex;
      flex-direction: column;
      justify-content: center;
      h3 {
        margin-left: 1rem;
        font-weight: 500;
        margin-top: 0;
      }
      .kubeapps-header-subtitle {
        margin-left: 1rem;
        img {
          max-height: 1rem;
          padding-right: 0.2rem;
          margin: 0;
        }
        span {
          color: var(--clr-p1-color, #666666);
          padding-top: 0.1rem;
        }
      }
    }
    .control-buttons {
      display: flex;
      align-items: center;
      justify-content: flex-end;
      height: 100%;
      .header-button {
        margin-left: 0.5rem;
      }
    }
  }
}

.kubeapps-header h1 {
  margin: 0;
  margin-left: 0.6rem;
  padding-top: 0.2rem;
}
