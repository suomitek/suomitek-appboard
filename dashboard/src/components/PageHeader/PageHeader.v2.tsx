import * as React from "react";

import "./PageHeader.v2.css";

function PageHeader(props: any) {
  return <header className="kubeapps-header">{props.children}</header>;
}

export default PageHeader;
