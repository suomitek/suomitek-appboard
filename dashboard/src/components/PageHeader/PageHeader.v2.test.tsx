import { shallow } from "enzyme";
import * as React from "react";
import PageHeader from "./PageHeader.v2";

it("should render a PageHeader", () => {
  const wrapper = shallow(
    <PageHeader>
      <h1>Title!</h1>
    </PageHeader>,
  );
  expect(wrapper).toMatchSnapshot();
  expect(wrapper.text()).toContain("Title!");
});
