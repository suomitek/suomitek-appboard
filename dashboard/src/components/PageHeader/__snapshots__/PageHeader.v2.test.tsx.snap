// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a PageHeader 1`] = `
<header
  className="kubeapps-header"
>
  <h1>
    Title!
  </h1>
</header>
`;
